storage/store.go

```diff
@@ -486,9 +489,15 @@ func (s *Store) Start() error {
 		s.gossip.RegisterCallback(capacityRegex, s.capacityGossipUpdate)
 	}
 
+	s.started = true
 	return nil
 }
 
+// Started returns whether the store is started completely.
+func (s *Store) Started() bool {
```

Comment:
Go-style isn't big on accessor methods. More idiomatic would be to name the field `Started` and access it directly. You'd only need the method of it was implementing an interface.